internal/api/db/db.go

```diff
@@ -0,0 +1,20 @@
+package db
+
+import (
+	"omshub/core-api/internal/api/db/models"
+
+	"gorm.io/driver/postgres"
+	"gorm.io/gorm"
+)
+
+func Init() *gorm.DB {
```

Comment:
The convention for constructor functions in Go is `New<Type>`, so I'd recommend calling this `NewDB`. 

`Init` in its lowercase form also has a special meaning in Go, so renaming this will be less confusing in the long run! https://tutorialedge.net/golang/the-go-init-function/